bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -78,14 +79,22 @@
             description = "If true, the last modified value will be included in the sitemap.")
     private static final String PROP_INCLUDE_LAST_MODIFIED = "include.lastmod";
 
-    @Property(label = "Change Frequency Properties",
+    @Property(label = "Change Frequency Properties", unbounded = PropertyUnbounded.ARRAY,
             description = "The set of JCR property names which will contain the change frequency value.")
     private static final String PROP_CHANGE_FREQUENCY_PROPERTIES = "changefreq.properties";
 
-    @Property(label = "Priority Properties",
+    @Property(label = "Priority Properties", unbounded = PropertyUnbounded.ARRAY,
             description = "The set of JCR property names which will contain the priority value.")
     private static final String PROP_PRIORITY_PROPERTIES = "priority.properties";
 
+    @Property(label = "DAM Folder Property",
+            description = "The JCR property name which will contain DAM folders to include in the sitemap.")
+    private static final String PROP_DAM_ASSETS_PROPERTY = "damassets.property";
+
+    @Property(label = "DAM Asset MIME Types", unbounded = PropertyUnbounded.ARRAY,
+            description = "MIME types allowed for DAM assets.")
+    private static final String PROP_DAM_ASSETS_TYPES = "damassets.types";
```

Comment:
Just thinking out loud... does this make sense to configure at the servlet level or the page level (next to the folders). I can't think of a reason why it would matter one way or the other, just thought I'd throw it out there.